### PR TITLE
docs: :memo: update with version 1.1 validation metrics

### LINKED ISSUE #74: Explore if GDM censoring can be simplified
If performance is similar with only end-of-pregnancy dates for censoring GDM, it would allow us to remove the maternal care visits to make the algorithm and data requirements simpler.

It would also streamline the algorithm if/when we eventually add support for using Medical Birth Register data to censor GMD instead of diagnoses from the National Patient Register, as both data sources would ultimately just provide one variable with date of the end of each pregnancy for each ID.

### LINKED ISSUE #107: Test simpler ways to de-duplication of HbA1c
In the original algorithm, old-unit HbA1c values were filtered out/deduplicated if a new-unit HbA1c value was present on the same date for the same individual.

This can perhaps be simplied to simply deduplicate all HbA1c values taken on the same date regardless of unit. This would align the algorithm more strictly with the recommended diagnostic practice (diabetes diagnosis required two elevated HbA1c values sampled on DIFFERENT dates).

Also mentioned in comment in #101

### PR DESCRIPTION
**Pull request by Aastedet**
Now includes simplified de-duplication of HbA1c and pregnancy definitions (see description in document) and the accuracy is slightly better overall, and particularly in the small, atypical groups (T1D old at diagnosis & T2D young at diagnosis). Very happy it turned out this way!

Closes #107, closes #74

### CODE REVIEW

File: vignettes/changes.Rmd [hunk 1]
```diff
@@ -20,26 +20,55 @@ these validations over the different versions.
...
 
 Algorithm validity across versions. Reports `PPV` (*positive predictive
 value*) and `sensitivity` for typical cases and cases with atypical age
 at onset of T1D (after age 40) and T2D (before age 40), respectively.
 
+### On pre-2019 data (as in the paper)
+
+**Overall and age at onset-stratified (paper table 1 & 2):**
+
+| Version | Diabetes type | PPV   | Sensitivity |
```
Comment (signekb) [inline vignettes/changes.Rmd:47]: Oh, do we want to do Semantic Versioning, so the first version (the paper version) is 1.0.0 and then this modified version is 1.0.1 (patch increase bc it's not adding functionality and it's only simplifying internal behaviour of the classifier)?

Comment (lwjohnst86) [reply vignettes/changes.Rmd:47]: I think the version refers to the algorithm, not the package. So semantic versioning might not exactly apply here.

**Review by lwjohnst86 — APPROVED** [review]
Looks good :) Only a small comment in relation to @signekb comment.

File: vignettes/changes.Rmd (hunk 1 above)
Comment (signekb) [reply vignettes/changes.Rmd:47]: That's true! Nvm ☺️

**Review by signekb — APPROVED** [review]

File: vignettes/changes.Rmd [hunk 2]
```diff
@@ -20,26 +20,55 @@ these validations over the different versions.
...
+    (originally, if a sampling time was available in the lab data, only
+    samples taken at the same time were de-duplicated)
 
 ## Validity
 
 Algorithm validity across versions. Reports `PPV` (*positive predictive
 value*) and `sensitivity` for typical cases and cases with atypical age
 at onset of T1D (after age 40) and T2D (before age 40), respectively.
 
+### On pre-2019 data (as in the paper)
```
Comment (signekb) [inline vignettes/changes.Rmd:43]: Maybe add an explicit comment on what the changes are from paper to 1.1?

**Review by signekb — COMMENTED** [review]
Very nice! Just one comment:

File: vignettes/changes.Rmd [hunk 3]
```diff
@@ -20,26 +20,55 @@ these validations over the different versions.
 
 ## Specific changes
 
-### Since the original published validation (version 1)
+### Since the original validation (version from the paper)
```
Comment (signekb) [inline vignettes/changes.Rmd:24]:
```suggestion

#### Version 1.1
```

**Pull request merged**